libsledge/src/memory_instructions.c

```diff
@@ -12,55 +12,55 @@ extern thread_local struct sledge_abi__wasm_module_instance sledge_abi__current_
 INLINE uint32_t
 instruction_memory_size()
 {
-	return (uint32_t)(sledge_abi__current_wasm_module_instance.memory.size / WASM_PAGE_SIZE);
+	return (uint32_t)(sledge_abi__current_wasm_module_instance.memory.size / (uint64_t)WASM_PAGE_SIZE);
```

Comment:
why the cast (uint64_t) ?  If the final return value is (uint32_t) I don't understand why (uint64_t) casting would be necessary. Maybe I'm missing something...

Reply:
It does look unnecessary I think. I'm just not sure if type mismatches might affect stuff like turning this division into a >> right shift since the denominator is a power of two. Mostly just me being paranoid.